env/production/replica_count.yaml

```diff
@@ -62,5 +62,5 @@ metadata:
   name: celery-hpa
   namespace: notification-canada-ca
 spec:
-  minReplicas: 6
-  maxReplicas: 6
+  minReplicas: 3
+  maxReplicas: 15
```

Comment:
We've got HPA with Celery pods in staging and this is not causing us problems even when doing load tests

https://github.com/cds-snc/notification-manifests/blob/9c16ca2993c2a22047061b2204868cd7f297c27f/env/staging/replica_count.yaml#L49-L56